src/Queries/Math/Add.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
... a similar blowup in code size here (though I feel less strongly about these).